src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/idam/cache/CachedIdamToken.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.idam.cache;
+
+public class CachedIdamToken {
+
+    public final String token;
```

Comment:
token -> accessToken

Reply:
I preferred generic name, for now it old version of the token, not open ID.

Reply:
changed